src/Update/RecipePatcher.php

```diff
@@ -203,7 +207,7 @@ private function getBlobPath(string $hash): string
         $hashStart = substr($hash, 0, 2);
         $hashEnd = substr($hash, 2);
 
-        return '.git/objects/'.$hashStart.'/'.$hashEnd;
+        return 'objects/'.$hashStart.'/'.$hashEnd;
```

Comment:
Small code simplification. Move `$originalFilesGitDir = trim($this->execute('git rev-parse --absolute-git-dir', $originalFilesRoot));` into THIS function, then return:

```php
return $originalFilesGitDir.'/objects/'.$hashStart.'/'.$hashEnd;
```

If you do this, then the two methods above need zero changes (their diffs can be reverted).

Reply:
Hmm, this seems not to work in case of subfolders.
I think the reason is that this getBlobPath() is called from two different places which are refering to different git paths:

- addMissingBlobs() is modifying the current project in $this->rootDir (git rev-parse resulting to either .git or ../.git/modules/projectName)
- generateBlobs() is manually generating the blobs for patch in temp location $tmpPath (git rev-parse always resulting to .git). 

I'll try a bit different approach simplifying the code